### PR TITLE
Testing practice

### PR DESCRIPTION
**Pull request by nriqu322**
This project was for unit testing in Javascript.

I build the following functions with their respective tests.
- Capitalize(string).
- Reverse(string).
- Calculator(sum, subtract, divide, and multiply).
- Caesar Cypher
- Array Analysis

### CODE REVIEW

**Comment by nriqu322** [comment]
Note that this project doesn't have a CSS file. Which is the reason why Stylint is failing.

File: src/caesarCypher.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  let result = [];
+  for (let i = 0; i < string.length; i += 1) {
+    if (upperCase(string[i])) {
+      result += String.fromCharCode(((string.charCodeAt(i) + num - 65) % 26) + 65);
+    } else {
+      result += String.fromCharCode(((string.charCodeAt(i) + num - 97) % 26) + 97);
+    }
+  }
+  return result;
+};
```
Comment (shshamim63) [inline src/caesarCypher.js:15]: The basic Caesar Cipher actually interact with the characters from [A-Z] &[a-z]. For example, if I perform an operation on the '("ATTACK.AT.DAWN",5)' should return me "FYYFHP.FY.IFBS". Right now the formation is returning me 'FYYFHP3FY3IFBS'

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (shshamim63) [inline .github/workflows/linters.yml:36]: As there is not css/ scss file, so you don't need to check the for stylint. Fell free to remove the following lines. And also stylintrc.json file.

File: src/caesarCypher.js (hunk 1 above)
Comment (nriqu322) [reply src/caesarCypher.js]: Fix function

File: .github/workflows/linters.yml (hunk 2 above)
Comment (nriqu322) [reply .github/workflows/linters.yml]: removed

**Pull request merged**